way_to_home/route/models.py

```diff
@@ -1 +1,73 @@
 """This module implements class that represents the route entity."""
+
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
+from way.models import Way
+from place.models import Place
+
+
+class Route(AbstractModel):
+    """Model for Route entity."""
+
+    way = models.ForeignKey(Way, on_delete=models.CASCADE)
+    start_place = models.ForeignKey(Place, on_delete=models.CASCADE)
+    end_place = models.ForeignKey(Place, on_delete=models.CASCADE)
+    time = models.TimeField()
+    transport_id = models.PositiveIntegerField()
+    position = models.PositiveSmallIntegerField()
+
+    def __str__(self):
+        """Method that returns route instance as string."""
+
+        return f'route from place with id: {self.start_place.id} ' \
+            f'to place with id {self.end_place.id} id'
```

Comment:
```suggestion
           return f'route from place with id: {self.start_place.id} ' \
            f'to place with id {self.end_place.id} id'
```
I think it is too long. For example, u can just write `from <id> to <id>`

Reply:
Done